src/state/loading/actions.js

```diff
@@ -7,4 +7,12 @@ export const toggleLoading = id => ({
   },
 });
 
+export const setLoading = (id, value) => ({
+  type: SET_LOADING,
+  payload: {
+    id,
+    value,
+  },
+});
+
```

Comment:
was it super necessary to add new function other than using `toggleLoading` ?

Reply:
There are few places that call the same async fetch at similar time - that makes initial loading value toggle twice or more, making toggle unreliable